AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -352,6 +352,7 @@ raz-zeitung.de,muensteraner-rechtsanwaelte.de,hardware-helden.de,naheland-apothe
 !
 ! NOTE: Regular rules
 !
+/audi/*/Bootstrap.js$domain=audi.com|audi.com.br|audi.ca|audi.ch|audi.co.za|audi.dk|audi.es|audi.ie|audi.it|audi.se|audi.com.tr|audi.de
```

Comment:
I'm not sure about blocking the script. Fixed in another way.
@AdguardTeam/filters-maintainers please take a look at this.